src/OrchardCore.Modules/OrchardCore.AdminMenu/Controllers/NodeController.cs

```diff
@@ -220,7 +220,7 @@ public async Task<IActionResult> Edit(AdminNodeEditViewModel model)
                 return NotFound();
             }
 
-            var editor = await _displayManager.UpdateEditorAsync(treeNode, updater: _updateModelAccessor.ModelUpdater, isNew: false);
+            var editor = await _displayManager.UpdateEditorAsync(treeNode, updater: _updateModelAccessor.ModelUpdater, isNew: false, "", "");
```

Comment:
Can you please use `String.Empty` instead of `""` in all these calls?

Reply:
2 seconds too late :)

Reply:
@ns8482e I'll fix that in dotnet format #12235